app-containers/kubectl/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.30.3
+CHKSUMS="sha512::303c823f74ccc5c1685ec16ba3de9dbbe9614ddf19e279e43ee392a4ef04ee68400a3e0c2a839d6a3f43712fab426084037ed871e3e4caafe8b37999fd83b73a"
+SRCS="https://github.com/kubernetes/kubernetes/archive/v$VER.tar.gz"
+ABTYPE=self
```

Comment:
This is for `autobuild/defines`, not spec.

Reply:
You should also add a CHKUPDATE field so that we can track and report updates.

See the [documentation](https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md) for instructions.